apps/web/components/connections/SourceCard.tsx

```diff
@@ -0,0 +1,258 @@
+import {VeniceProvider} from '@usevenice/engine-frontend'
+import {DropdownMenuPrimitive as DropdownMenu} from '@usevenice/ui'
```

Comment:
Use radix components directly for now; once we have a couple use cases, we can build a shared component into `@usevenice/ui` rather than pre-maturely abstract it.